src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
I do not understand ... which other rules would have this problem? It's not an issue with `beforeBlockString`, but an issue with the location of the warning for this particular rule.